### PR TITLE
fix/tabBarSizeOnHideMode

### PR DESCRIPTION
**Pull request by andryuwka**
fixed size of presented view when tab bar is hidden

### CODE REVIEW

**Comment by andreamazz** [comment]
Doesn't this break the code when the tab bar is translucent?

File: Source/ScrollingNavbar+Sizes.swift [hunk 1]
```diff
@@ -20,9 +20,13 @@ extension ScrollingNavigationController {
...
 
     var tabBarOffset: CGFloat {
-        // Only account for the tab bar if a tab bar controller is present and the bar is not translucent
+        // Only account for the tab bar if a tab bar controller is present and the bar is not hidden
         if let tabBarController = tabBarController {
-            return tabBarController.tabBar.translucent ? 0 : tabBarController.tabBar.frame.height
+            if tabBarController.tabBar.hidden {
+                return 0
+            } else {
+                return tabBarController.tabBar.translucent ? 0 : tabBarController.tabBar.frame.height
```
Comment (andryuwka) [inline Source/ScrollingNavbar+Sizes.swift]: @andreamazz Oh, excuse me) i got it and fixed

Comment (andryuwka) [inline Source/ScrollingNavbar+Sizes.swift]: Hmm, yes, it's break code when tab bar is translucent, but in old version it's break view when tab bar is hidden and not translucent; appears black rect at bottom. i'm think how fix this now.

**Comment by andreamazz** [comment]
Uhm, I'm having an hard time understanding the issue, anyhoo, since you reverted like the start, I can't merge this.

Can I just add the same check that if the tabbar is either hidden or translucent the offset is 0? Would that do?

**Pull request closed without merging**